docs/contribute/new-provider.rst

```diff
@@ -8,8 +8,8 @@ The actual implementation gives two providers: :ref:`grid5000` and
 then implementing a new one is easy as 500 lines of Python code.
 
```

Comment:
Maybe fixe the above sentence : 
`
The actual implementation gives you several providers, ...
`